plugins/community/neoeinstein-tonic/v0.4.1/buf.plugin.yaml

```
@@ -0,0 +1,22 @@
+version: v1
+name: buf.build/community/neoeinstein-tonic
+plugin_version: v0.4.1
+source_url: https://crates.io/crates/protoc-gen-tonic
+description: Generates Tonic gRPC server and client code using the Prost! code generation engine.
+deps:
+  - plugin: buf.build/community/neoeinstein-prost:v0.3.1
```

I think this ought to bump to `v0.4.0` to grab the new prost version, above?

```suggestion
  - plugin: buf.build/community/neoeinstein-prost:v0.4.0
```